.env.example

```
@@ -11,3 +11,7 @@ BE_IMAGE_NAME=
 # ---------- Frontend  ----------
 FE_IMAGE=
 FE_IMAGE_NAME=
+
+# ---------- Crawler -----------
+BE_API_URL=
+CRON_SCHEDULE=
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)